### PR TITLE
Update registry client v2

### PR DESCRIPTION
**Pull request by gabomarf**
Fixes https://github.com/doitintl/kube-secrets-init/issues/22
Fixes https://github.com/doitintl/kube-secrets-init/issues/24 

fix error linet from sagikazarmark

### CODE REVIEW

**Comment by gabomarf** [comment]
@alexei-led  can you review this pull request, please?  this pull request  i have done to fix error linet from @sagikazarmark sagikazarmark (https://github.com/doitintl/kube-secrets-init/pull/30)  

I need this change please friend

File: cmd/secrets-init-webhook/main.go [hunk 1]
```diff
@@ -11,7 +12,7 @@ import (
 	"github.com/Masterminds/semver/v3"
 
 	"github.com/doitintl/kube-secrets-init/cmd/secrets-init-webhook/registry"
-	"github.com/pkg/errors"
+	_ "github.com/pkg/errors"
```
Comment (sagikazarmark) [inline cmd/secrets-init-webhook/main.go:15]: This doesn't really make sense if it's imported again above.

**Comment by sagikazarmark** [comment]
Why don't you tell me what the proper fix is instead? This is certainly not the appropriate way of resubmitting my original PR since it's not abandoned.

**Comment by sagikazarmark** [comment]
Looks like updating the Go version was the solution. Builds pass now in the original PR.

**Comment by gabomarf** [comment]
@sagikazarmark I'm sorry, it's true that I should have written to you... , but seeing that your "pull request" remained there without movement, I wanted to press, anyway we can close the pull request that it generates and continue with yours, I'm really sorry, go into desperation mode to occupy something cleaner

**Pull request closed without merging**